README

```diff
@@ -1,12 +1,12 @@
-00000000011111111112222222222333333333344444444445555555555666666666677777777778
+﻿00000000011111111112222222222333333333344444444445555555555666666666677777777778
 12345678901234567890123456789012345678901234567890123456789012345678901234567890
        ______ ___ __   _  _____ _    _       ______  _____ ___ _____ _   _
        |_____  |  | \  | |  ___ |    | |     |____| |____/  |    |    \_/  
        _____| _|_ |  \_| |____| |____| |____ |    | |   \_ _|_   |     |   
                                                _  _ _ ____ _  _ ____ ____
                                                 \/  | |=== |/\| |=== |--<
 
-Sin-gu-la-ri-ty (noun) - a distinctive feature, a uniqueness; a point at which
+Sin·​gu·​lar·​i·​ty (noun) - a distinctive feature, a uniqueness; a point at which
```

Comment:
This one is weird for me, because I pronounce it Sin Gu La Ri Ty or Sin Gu Lar Ri ty.

Reply:
That's something to take up with Merriam-Webster https://www.merriam-webster.com/dictionary/singularity